### PR TITLE
Update docs with --EXTENSIONS-- information

### PR DESCRIPTION
**Pull request by MaxSem**

### CODE REVIEW

File: write-test.php [hunk 1]
```diff
@@ -372,29 +391,17 @@ function or an array function) , or a function provided by one of PHP's numerous
...
 --TEST--
-Check for exif_read_data, unusual IFD start
+Check for libsodium argon2i
+--EXTENSIONS--
+sodium
 --SKIPIF--
-&lt;?php
-	if (!extension_loaded('exif')) print 'skip exif extension not available';
-?&gt;
+<?php
```
Comment (cmb69) [inline write-test.php:401]: Not sure why you have replaced that test case, but that `<?php` needs to be escaped (otherwise the code would run).

Comment (MaxSem) [reply write-test.php]: I've replaced it because the old one doesn't have a skipif anymore:)

**Review by cmb69 — APPROVED** [review]
Ah, I see. Makes sense.

**Review by carusogabriel — APPROVED** [review]

**Comment by MaxSem** [comment]
Can someone merge this, please? :)

**Comment by cmb69** [comment]
I have merged this PR now. We need to update phpt_details.php as well, and there it should be made clear that `--EXTENSIONS--` is only available as of PHP 8.1.0, and that for versions prior to that respective `--SKIPIF--` clauses need to be used instead.

**Pull request merged**